tools/CleanOutput.py

```diff
@@ -84,13 +84,23 @@ def parse_args():
         action='count',
         default=0,
         help="Verbosity (-v, -vv, ...)")
+    parser.add_argument(
+        '--force',
+        action='store_true',
+        help="Suppress all errors and warnings")
     return parser.parse_args()
 
 
 if __name__ == "__main__":
     args = parse_args()
 
-    # Set the log level
-    logging.basicConfig(level=logging.WARNING - args.verbose * 10)
+    if args.force:
+        logging.basicConfig(level=1000)
```

Comment:
Can we add this feature in a way that doesn’t disable logging? How about you just pass the `force` flag to the `clean_output` function and  use it in line 56 by replacing the `else` with `elif not force`? I think that‘ll do what you want, and still provide useful output.

You can then also add the `-v` flag to where you call the script, so the log will report „Removed file ...“ messages.